iOS/Flows/Conversation/_New/ConversationViewControllerDataSource.swift

```diff
@@ -17,7 +17,6 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
 
     enum SectionType: Hashable {
         case conversation(ChannelId)
-        case loadMore
```

Comment:
With the new message order, there's no need for a separate load more section. I can just tack the load more cell at the end without screwing up message indices.

Reply:
Gotta love a one liner like this. So many benefits.